Filters:
--- Current
+++ TO BE APPLIED
@@ -1,84 +1,95 @@
 * Criteria:
-    to: someone-else@gmail.com
+    from: spammer2
   Actions:
-    archive
-    mark as important
-    never mark as spam
-    mark as read
-    star
-    categorize as: social
-    forward to: forward-address@gmail.com
+    delete
 
 * Criteria:
-    query: is:muted
+    query: "buy this thing"
   Actions:
-    archive
-    mark as important
-    never mark as spam
-    mark as read
-    star
-    categorize as: social
-    forward to: forward-address@gmail.com
+    delete
 
 * Criteria:
     query: 
-      cc:peeker@yahoo.com
-      -subject:"a subject"
+      list:foobaz.mail.com
+      -"action needed"
+  Actions:
+    delete
+
+* Criteria:
+    to: alias@gmail.com
+  Actions:
+    categorize as: promotions
+
+* Criteria:
+    from: baz+zuz@mail.com
+  Actions:
+    mark as important
+    categorize as: social
+    forward to: other@mail.com
+
+* Criteria:
+    from: notfriend@gmail.com
+    subject: "hey there"
+    query: -to:none@gmail.com
   Actions:
     archive
-    mark as important
-    never mark as spam
-    mark as read
     star
-    categorize as: social
-    forward to: forward-address@gmail.com
+    categorize as: forums
 
 * Criteria:
-    from: someone@gmail.com
+    to: pippo+spammy@gmail.com
+  Actions:
+    delete
+
+* Criteria:
+    query: bcc:aaaa@gmail.com
+  Actions:
+    categorize as: updates
+
+* Criteria:
+    query: 
+      list:{
+        list3
+        list1
+        list4
+        list6
+      }
+      -to:none@gmail.com
+  Actions:
+    apply label: differentlabel
+
+* Criteria:
+    from: spammer1
+    subject: "spam mail"
+    query: 
+      cc:foo@baz.com
+      bcc:bar@baz.com
+  Actions:
+    delete
+
+* Criteria:
+    query: 
+      list:{
+        list3
+        list1
+        list4
+        list6
+      }
+      -to:none@gmail.com
   Actions:
     archive
-    mark as important
-    never mark as spam
-    mark as read
-    star
-    categorize as: social
-    forward to: forward-address@gmail.com
+    categorize as: personal
+    apply label: maillist
 
 * Criteria:
-    query: bcc:bccer@gmail.com
+    query: 
+      list:{
+        list3
+        list1
+        list4
+        list6
+      }
+      -to:none@gmail.com
   Actions:
-    archive
-    mark as important
-    never mark as spam
-    mark as read
-    star
-    categorize as: social
-    forward to: forward-address@gmail.com
+    apply label: thirdlabel
 
-* Criteria:
-    query: replyto:replyer@gmail.com
-  Actions:
-    archive
-    mark as important
-    never mark as spam
-    mark as read
-    star
-    categorize as: social
-    forward to: forward-address@gmail.com
-
-* Criteria:
-    query: "something in the body"
-  Actions:
-    archive
-    mark as important
-    never mark as spam
-    mark as read
-    star
-    categorize as: social
-    forward to: forward-address@gmail.com
-
-* Criteria:
-    query: list:maillist@google.com
-  Actions:
-    never mark as important
-

Labels:
--- Current
+++ TO BE APPLIED
@@ -1,3 +1,4 @@
-label4; color: white, black
-label2; color: red, blue
-label3
+label4; color: white, gray
+maillist
+thirdlabel
+differentlabel
